app/sentry.php

```diff
@@ -22,11 +22,11 @@
 
 /** @var NULL|Raven_Client $client */
 $client = NULL;
-$stage = $appConfig['parameters']['stage'];
+$stage = $appConfig['parameters']['stage'] ?? 'unknown';
```

Comment:
domluvte se pro jakou verzi php to píšete, v jednom z commitů Matěj upravoval `?:` na `::`

Tenhle commit vůbec nepatří do tohohle PR